pyproject.toml

```
@@ -8,7 +8,7 @@ name = "pylint"
 description = "python code static checker"
 readme = "README.rst"
 keywords = [ "lint", "linter", "python", "static code analysis" ]
-license = { text = "GPL-2.0-or-later" }
+license-files = [ "LICENSE", "CONTRIBUTORS.txt" ]
```

```suggestion
license = "GPL-2.0-or-later"
license-files = [ "LICENSE", "CONTRIBUTORS.txt" ]
```
Not sure if required.